pkg/volume/empty_dir/empty_dir.go

```diff
@@ -86,7 +87,7 @@ func (plugin *emptyDirPlugin) newBuilderInternal(spec *volume.Spec, pod *api.Pod
 		mountDetector: mountDetector,
 		plugin:        plugin,
 		rootContext:   opts.RootContext,
-		chconRunner:   chconRunner,
+		chconRunner:   chconrunner,
```

Comment:
I have the same question, @swagiaal.  I don't think emptyDir needs a chcon runner anymore.  It will need to pass the `rootcontext` argument to mount, but the kubelet can now do the chcon.